gadget-container/entrypoint.sh

```diff
@@ -156,6 +156,15 @@ fi
 
 ## Hooks Ends ##
 
+# Create symlinks for tools according to the avaibility of BTF
+if grep -q CONFIG_DEBUG_INFO_BTF=y /host/boot/config-$(uname -r); then
```

Comment:
Could you directly check if `/sys/kernel/btf/vmlinux` exists? So it would work regardless of the files in `/boot/config-*`.

Reply:
Makes sense. Updated.